pkg/generator/helper/helper.go

```diff
@@ -420,12 +428,12 @@ func MetadataFiles(sourceDirectory string) ([]string, error) {
 
 func stepTemplate(myStepInfo stepInfo) []byte {
 
-	funcMap := template.FuncMap{
-		"flagType":   flagType,
-		"golangName": golangNameTitle,
-		"title":      strings.Title,
-		"longName":   longName,
-	}
+	funcMap := sprig.HermeticTxtFuncMap()
```

Comment:
do we need the additional dependency here?
Wouldn't below code work with standard template and `FuncMap` as well?